### DISCUSSION TITLE
terminal name

### DISCUSSION THREAD

**Discussion by darkforest42** [mason-org/mason.nvim#318] [Q&A]
what's your terminal tool? I want to download it.
another question, how to display select language window? why is my window style different yours
![image](https://user-images.githubusercontent.com/7768408/185759643-630e8077-2a88-4beb-a481-76145ebb0d80.png)

**Comment by williamboman** [comment]
Hello! I use:

- kitty (terminal)
- kanagawa.nvim (colorscheme)
- dressing.nvim (for nicer `vim.ui` interfaces)

**Reply by darkforest42** [reply]
thank you!
